pom.xml

```diff
@@ -138,10 +142,13 @@
         </executions>
       </plugin>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Minor tweak: You can remove this groupId for the plugin. "org.apache.maven.plugins" is the default groupId for plugins.

Reply:
Thank you! In: 2ec3d5d